staff/views/applications.py

```diff
@@ -31,10 +32,14 @@ def post(self, request, *args, **kwargs):
         wizard = Wizard(application, form_specs)
         pages = list(wizard.get_pages())
 
+        review_time = timezone.now()
+
         for page in pages:
             form = page.get_bound_approval_form(request.POST)
             if form.instance.pk:
                 form.save()
+                page.instance.last_reviewed_at = review_time
```

Comment:
At the moment, each page has `last_reviewed_at`.

But, because the review is currently for the entire form instead of per page, not sure there's a semantic way to distinguish between pages reviewed and not reviewed :thinking:

Reply:
Actually, "approved_by" can be null.